pykafka/utils/struct_helpers.py

```diff
@@ -68,6 +70,7 @@ def _unpack(fmt, buff, offset, count=1):
     """
     items = []
     array_fmt = None
+    buff = get_bytes(buff)
```

Comment:
Same thing here, and in L49 too: if the tests expect this to accept unicode, it's the tests that need fixing.